src/variables/body/index.ts

```diff
@@ -40,7 +40,9 @@ class Description extends Widget {
 
     this.model.currentChanged.connect(
       (model: Variables.IModel, variable: Variables.IVariable) => {
-        this.node.innerHTML = this.renderDescription(this.model.current);
+        this.node.innerHTML = this.renderDescription(
```

Comment:
Updating the DOM of this widget should happen in an override of `protected onUpdateRequest` and you should only need to call `this.update();`.

For an example, consider this one: https://github.com/jupyterlab/jupyterlab_apod/blob/1.0/src/index.ts#L58-L85

Although in our case, we don't need `onUpdateRequest` to be `async`. It can be simply `protected`.